lib/core/config.js

```diff
@@ -222,8 +232,29 @@ Config.prototype.loadContractsConfigFile = function() {
 
   let configFilePath = this._getFileOrOject(this.configDir, 'contracts', 'contracts');
 
-  const newContractsConfig = this._mergeConfig(configFilePath, configObject, this.env);
-
+  let newContractsConfig = this._mergeConfig(configFilePath, configObject, this.env);
+  if (newContractsConfig.gas != "auto") {
+      newContractsConfig.gas = utils.getWeiBalanceFromString(newContractsConfig.gas, web3);
+  }
+  if ('deployment' in newContractsConfig) {
```

Comment:
I *believe* that there should be always a `deployment` object in the config. Otherwise the config is broken.

Looping in @andremedeiros to confirm this.